x-pack/plugins/security_solution/scripts/endpoint/common/roles_users/with_artifact_read_privileges_role.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Role } from '@kbn/security-plugin/common';
+import { getNoResponseActionsRole } from './without_response_actions_role';
+
+export const getWithArtifactReadPrivilegesRole: () => Omit<Role, 'name'> = () => {
+  const noResponseActionsRole = getNoResponseActionsRole();
+  return {
+    ...noResponseActionsRole,
+    kibana: [
+      {
+        ...noResponseActionsRole.kibana[0],
+        feature: {
+          ...noResponseActionsRole.kibana[0].feature,
+          siem: [
+            'minimal_all',
+            'blocklist_read',
+            'trusted_applications_read',
+            'host_isolation_exceptions_read',
+            'event_filters_read',
```

Comment:
Consider sorting out these privileges alphabetically. Also in other roles, if you do change it here. It would be easier to add new privileges later. I had done something similar in my PR with licenses.

Reply:
For now, I'd merge it this way, because my next (draft) PR (https://github.com/elastic/kibana/pull/151046) builds on this one. There, I ordered the privileges in a similar fashion as they are in our roles sheet, but I'm open to order them alphabetically, too, it was just easier to see them this way.

I added you as a reviewer, so we can return to this question there.